test_scripts/API/SetAudioStreamingIndicator/037_ATF_N_TC_SetAudioStreamingIndicator_USER_DISALLOWED_media_app_RPC_disallowed_by_user.lua

```diff
@@ -0,0 +1,245 @@
+-- Requirement summary:
+-- [USER_DISALLOWED]: Conditions for USER_DISALLOWED result code
+-- [MOBILE_API] SetAudioStreamingIndicator
+-- [HMI_API] [MOBILE_API] AudioStreamingIndicator enum
+-- [PolicyTable] SetAudioStreamingIndicator RPC
+--
+-- Description:
+-- For Genivi applicable ONLY for 'EXTERNAL_PROPRIETARY' Polciies
+-- SDL must return 'USER_DISALLOWED, success:false' to mobile app
+-- in case the requested RPC is included to the group disallowed by the user.
+--
+-- 1. Used preconditions
+-- SDL is built with External Proprietary flag.
+-- Include in Testing_group(user_consent group) RPC SetAudioStreamingIndicator
+-- Register and activate media application
+-- Perform successfull PTU for External Proprietary flow
+-- Disallow by user SetAudioStreamingIndicator
+--
+-- 2. Performed steps
+-- Send SetAudioStreamingIndicator(audioStreamingIndicator = "PAUSE")
+--
+-- Expected result:
+-- SDL->mobile: SetAudioStreamingIndicator_response("USER_DISALLOWED", success:false)
+---------------------------------------------------------------------------------------------
+
+--[[ General configuration parameters ]]
+config.deviceMAC = "12ca17b49af2289436f303e0166030a21e525d266e209267433801a8fd4071a0"
+config.application1.registerAppInterfaceParams.appHMIType = {"MEDIA"}
+config.application1.registerAppInterfaceParams.isMediaApplication = true
+
+--[[ Required Shared libraries ]]
+local commonFunctions = require ('user_modules/shared_testcases/commonFunctions')
+local commonSteps = require('user_modules/shared_testcases/commonSteps')
+local commonPreconditions = require('user_modules/shared_testcases/commonPreconditions')
+local testCasesForPolicyTable = require('user_modules/shared_testcases/testCasesForPolicyTable')
+local json = require('json')
+
+--[[ Local Functions ]]
+
+--[[@create_ptu_SetAudioStreamingIndicator_group: update preloaded_pt and creates 2 PTU files
+! preloaded_pt.json: add Testing_group with RPC SetAudioStreamingIndicator and user consent
+! SetAudioStreamingIndicator_group1.json: Assign groups "Base-4", "Testing_group" to application
+! SetAudioStreamingIndicator_group1.json: Assign only group "Base-4" to application
+! @parameters: NO
+]]
+local function create_ptu_SetAudioStreamingIndicator_group()
+  commonPreconditions:BackupFile("sdl_preloaded_pt.json")
+  os.execute(" cp " .. config.pathToSDL .. "sdl_preloaded_pt.json" .. " " .. "SetAudioStreamingIndicator_group1.json" )
+  os.execute(" cp " .. config.pathToSDL .. "sdl_preloaded_pt.json" .. " " .. "SetAudioStreamingIndicator_group2.json" )
+  local pathToFile = config.pathToSDL .. 'sdl_preloaded_pt.json'
+  local file = io.open(pathToFile, "r")
+  local json_data = file:read("*all")
+  file:close()
+
+  local data = json.decode(json_data)
+  if(data.policy_table.functional_groupings["DataConsent-2"]) then
+    data.policy_table.functional_groupings["DataConsent-2"].rpcs = { json.null }
```

Comment:
@istoimenova in such kind of assignment you will get in json file:

> "DataConsent-2": {
> 				"rpcs": [null],
> 				"user_consent_prompt": "DataConsent"
> 			},

For getting this group as was in original preloaded file 

>             "DataConsent-2": {
>                 "user_consent_prompt": "DataConsent",
>                 "rpcs": null
>             },

please use

> data.policy_table.functional_groupings["DataConsent-2"].rpcs =  json.null